src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,375 @@
+
+The following schema illustrates each of these data types in use:
+
+```ts
+import { z, defineCollection } from 'astro:content';
+
+defineCollection({
+  schema: {
+    isDraft: z.boolean(),
+    title: z.string(),
+    sortOrder: z.number(),
+    image: z.object({
+      src: z.string(),
+      alt: z.string(),
+    }),
+    tags: z.array(z.string()), // An array of strings
+    footnote: z.string().optional(),
+    author: z.string().default('Anonymous'),
+    language: z.enum(['en', 'es']),
+  }
+})
+```
+
+### Advanced schema features
+
+You can use all of Zod’s properties and methods with content schemas. This includes transforming a frontmatter value into another value, checking the shape of string values with built-in regexes, and more.
+
+```ts
+{
+  // Allow only strings representing email addresses
+  authorContact: z.string().email(),
+  // Allow URL strings only (e.g. `https://example.com`)
+  canonicalURL: z.string().url(),
+  // Parse publishDate as a browser-standard `Date` object
+  publishDate: z.string().transform(str => new Date(str)),
+}
+```
+
+See [Zod’s documentation](https://github.com/colinhacks/zod) for a complete list of features.
+
+## Querying content collections
+
+Astro provides two functions to query collections:
+
+### `getCollection()`
+
+`getCollection()` returns multiple entries in a collection. It requires the name of a `collection` as a parameter.  By default, it returns all items in the collection.
+
+It can also take a second, optional parameter: a filter function based on schema properties. This allows you to query for only some items in a collection based on `id`, `slug`, or frontmatter values via the `data` object.
+
+```astro
+---
+import { getCollection } from 'astro:content';
+// Get all `src/content/blog/` entries
+const allBlogPosts = await getCollection('blog');
+// Only return posts with `draft: true` in the frontmatter
+const draftBlogPosts = await getCollection('blog', ({ data }) => {
+  return data.draft === true;
+});
+---
+```
+
+### `getEntry()`
+
+`getEntry()` is function that returns a specific entry in a collection by entry ID (file path relative to the collection). Both of these are required parameters.
+
+```astro
+---
+import { getEntry } from 'astro:content';
+
+const enterprise = await getEntry('blog', 'enterprise.md');
+---
+```
+
+### Data returned from a collections query
+
+`getCollection()` and `getEntry()` will return entries that include:
+ - `id` - a unique ID using the file path relative to `src/content/[collection]`
+ - `slug` - a URL-ready slug. Defaults to the ID without the file extension.
+ - `data` - an object of frontmatter properties inferred from your collection schema. Defaults to `any` if no schema is configured.
+ - `body` - a string containing the raw body of the Markdown or MDX document.
+
+### Landing page example
+ 
+Given the following collection:
```

Comment:
I actually moved the schema sentence into the lead-in after removing that redundant file tree. Resolving since I think this sentence is covered by inline comments on the code sample?